components/UseCaseCard.vue

```diff
@@ -22,13 +35,37 @@
 
 export default {
   props: {
-    panel: {
+    title: {
+      type: String,
+      default: null,
+      required: false
+    },
+    description: {
+      type: String,
+      default: null,
+      required: false
+    },
+    usecase: {
       type: Array,
       default: null,
       required: false
     }
-  },
-  mounted () {
   }
 }
 </script>
+
+<style>
+.vueperslide__image {
+  background-size: contain;
+  background-repeat: no-repeat;
+  height: 300px;
```

Comment:
Is the fixed height needed here (since it's also already in the 'vueper slides' tag, it might cause trouble if you change one but not the other).

Reply:
Good point! Yes, it can be removed. Just did some testing. Also the fixed height in the vueper slides can be removed, too. Just keep the aspect-ratio will be fine.